charts/metering-ansible-operator/values.yaml

```diff
@@ -222,17 +221,20 @@ olm:
 
       ### Before you start
 
-      Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores. At least one Node should have 2GB of RAM and 2 CPU cores. Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+      Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores.
+      At least one Node should have 2GB of RAM and 2 CPU cores.
+      Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+
+      Metering requires configuring storage, pleaese review the [configuring persistent storage documentation](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html) before proceeding.
 
       ### Common Configurations
 
-      Metering works out of the box without any customizations or configuration. Read the [full documentation](https://github.com/operator-framework/operator-metering/blob/master/Documentation/metering-config.md) for more details.
+      * **Store data in object storage or in a PersistentVolume** - Store your report output [in an object storage bucket or in a PersistentVolume](https://docs.openshift.com/container-platform/4.2/metering/metering-configure-persistent-storage.html).
 
```

Comment:
This link is one level down:
https://docs.openshift.com/container-platform/4.2/metering/configuring-metering/metering-configure-persistent-storage.html